openquake/commonlib/calculators/event_based_risk.py

```diff
@@ -74,19 +80,37 @@ def event_based_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
     return acc
 
 
-def mean_quantiles(quantiles):
+def _mean_quantiles(quantiles):
     yield 'mean'
     for q in quantiles:
         yield 'quantile-%s' % q
 
 
-def loss_map_names(conditional_loss_poes):
+def _loss_map_names(conditional_loss_poes):
     names = []
     for clp in conditional_loss_poes:
         names.append('poe~%s' % clp)
     return names
 
 
+def extract_avglosses(dstore, pattern):
+    """
+    Function extracting the average losses from the loss curves
+    for each realization.
+
+    :param dstore: a datastore object
+    :param pattern: '/loss_curves-rlzs/%s' or '/agg_loss_curve-rlzs/%s'
```

Comment:
If not, one immediately gets a clear KeyError from a couple of lines below. So, it is okay as it is, IMO.